<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-E5V544TN" name="com.gmail.testcases.SikuliSignIn" tests="1" failures="1" timestamp="17 Jul 2018 11:28:00 GMT" time="43.506" errors="0">
  <testcase name="login_logout_test" time="43.506" classname="com.gmail.testcases.SikuliSignIn">
    <failure type="org.sikuli.script.FindFailed" message="C:/Workspace/GmailSignUp/sikuli_images/btn_Next.png: (93x43) seen at (777, 549) with 0.94 in S(0)[0,0 1366x768] E:Y, T:3.0">
      <![CDATA[FindFailed: C:/Workspace/GmailSignUp/sikuli_images/btn_Next.png: (93x43) seen at (777, 549) with 0.94 in S(0)[0,0 1366x768] E:Y, T:3.0
  Line 2759, in file Region.java

	at org.sikuli.script.Region.wait(Region.java:2759)
	at org.sikuli.script.Region.find(Region.java:2336)
	at org.sikuli.script.Region.getLocationFromTarget(Region.java:3213)
	at org.sikuli.script.Region.click(Region.java:3698)
	at org.sikuli.script.Region.click(Region.java:3683)
	at com.gmail.testcases.SikuliSignIn.login_logout_test(SikuliSignIn.java:69)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- login_logout_test -->
</testsuite> <!-- com.gmail.testcases.SikuliSignIn -->
